lib/libutee/include/utee_defines.h

```diff
@@ -78,6 +79,9 @@ static inline uint32_t __tee_alg_get_class(uint32_t algo)
 	if (algo == TEE_ALG_RSAES_PKCS1_OAEP_MGF1_MD5)
 		return TEE_OPERATION_ASYMMETRIC_CIPHER;
 
+	/* Implementation defined algorithm IDs must be specifically handled */
+	assert(((algo >> 28) & 0xF) != 0xF);
```

Comment:
Can we return `__TEE_OPERATION_UNDEFINED`  (with the value 0) or such instead of asserting?